schematics/types/base.py

```diff
@@ -719,6 +730,9 @@ def to_native(self, value, context=None):
             return value
 
         for fmt in self.formats:
+            if fmt == self.TIMESTAMP_FORMAT:
+                return arrow.get(value)
```

Comment:
Let's not add a whole new dependency for such a simple conversion. `datetime.utcfromtimestamp(float(value))` does the same thing. Also, this should look out for exceptions and ignore, in case some other format matches the input.